.github/workflows/skribenten.yml

```diff
@@ -45,7 +45,7 @@ jobs:
           ./gradlew :pensjon-brevbaker-api-model:publishToMavenLocal
           ./gradlew :alder-brevbaker-api-model:publishToMavenLocal
           ./gradlew :ufoere-brevbaker-api-model:publishToMavenLocal 
-          ./gradlew :skribenten-backend:build
+          ./gradlew :skribenten-backend:build installDist
```

Comment:
Er ikke `installDist` her for alle modulene? Må du ikke namespace den til skribenten?

Reply:
Øøh, jo.